Cargo.toml

```diff
@@ -41,7 +41,7 @@ wasm-bindgen-test = { version = "0.3" }
 
 [features]
 all = ["bincode", "ini", "json", "pretty", "ron", "toml", "yaml"]
-default = []
+default = ["all"]
```

Comment:
Should probably keep the default features to [] to avoid surprises when most users probably just want to update to the latest bevy.

Reply:
I agree. `all` feature is not active by default to reduce binary size and compile times by reducing the number of dependencies. It's meant to be used for prototyping or getting started for a quick demo, it's not for production.

Reply:
Oh my fault, I added this for test.